controllers/datadogagent/override/global.go

```diff
@@ -15,6 +18,13 @@ import (
 func ApplyGlobalSettings(manager feature.PodTemplateManagers, config *v2alpha1.GlobalConfig) *corev1.PodTemplateSpec {
 	// TODO(operator-ga): implement ApplyGlobalSettings
 
+	if config != nil && config.Kubelet != nil && config.Kubelet.TLSVerify != nil {
```

Comment:
fine for now, but when we add call to default DatadogAgent then globalConfig shouldn't be `nil`